doxygen-awesome-interactive-toc.js

```diff
@@ -55,9 +55,7 @@ class DoxygenAwesomeInteractiveToc {
                         headerNode: document.getElementById(id)
                     })
 
-                    document.getElementById("doc-content")?.addEventListener("scroll", () => {
-                        DoxygenAwesomeInteractiveToc.update()
-                    })
+                    document.getElementById("doc-content")?.addEventListener("scroll",throttle(DoxygenAwesomeInteractiveToc.update, 100))
```

Comment:
The "this" keyword is missing before the throttle method. Without this, the fix does not work and causes errors in the console.

Reply:
Updated the code, please review and let me know if it works now.